core/src/modules/validators/isEmail.ts

```diff
@@ -5,6 +5,6 @@
  * We cannot use import isEmail from "validator/lib/isEmail" until https://github.com/validatorjs/validator.js/pull/1435 is merged in.
  */
 export default function isEmail(string: string) {
```

Comment:
This comment is waiting on something to be merged that has been merged at this point. We have `validator` installed... could we just use that?

If not, maybe this comment should be removed

Reply:
That'd be an @bleonard call.

Reply:
I'd remove that comment - we still don't really want to use the more strict email validator from validator.js - they won't like the leading/trailing spaces